rfcs/0005-pablo-distribution-assets/0005-pablo-distribution.adoc

```diff
@@ -498,8 +498,34 @@ endwhile (no)
 :save(rewards_pool);
 stop
 ----
+==== Reward Pool Governance
 
-==== Update Reward Pool
+===== Update Reward Allocation Per Pool
+
+Each reward pool would have its own reward pot account.
+
+* sub-accounts for the rewards pots,
+** A prefix is needed to differentiate between the pallet "hot wallet" which stores the claimable funds and the "cold wallet" which transferred provided treasury funds.
+... Hot wallet prefix: "Hot"
+... Cold wallet prefix: "Cold"
+* Lock the assets in the cold wallet so that funds can be transferred only when unlocked (defensive programming)
```

Comment:
not sure. you can lock funds without having hot or cold. via lock. it will lock some amount. also there is reserve. i mean not sure why 2 wallets are needed? what mechanics it helps, except defensive programming?

Reply:
it provides a distinction between the claimable funds and the currently locked funds - an extra layer between the users/claiming and the locked/held funds.

Reply:
if you think that just locking the funds in one wallet is fine, then we can discuss it  with @vimukthi-git

Reply:
I'm ok with just locking in a single account, cold/hot wallet concept is not needed for derived pallet accounts as they accessed only programmatically as I understand more.

Reply:
Updated in https://github.com/ComposableFi/composable/pull/1473/commits/fcaddfc09bfea06e1f463ab55ea340f1afd47c12